.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: node_js
+node_js:
+  - "7"
+  - "8"
```

Comment:
The build for node 8 does not seem to work, could we remove it for now or see why it does not work?